pyapp/cookiecutter.json

```diff
@@ -14,6 +14,7 @@
     "devdata": ["no", "yes"],
     "postgres": ["no", "yes"],
     "docker": ["no", "yes"],
+    "__postgres_version": "15.3-alpine",
```

Comment:
Oops, the `__postgres_version` option wasn't actually working because it was missing from all three `cookiecutter.json`'s